active/NumberBox/NumberBox.md

```diff
@@ -0,0 +1,184 @@
+
+## Examples
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+  
+<!-- Code samples should be in C# and/or C++/WinRT -->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+
+### Create a NumberBox
+
+XAML
+```XAML
+<NumberBox AutomationProperties.Name="simple NumberBox"/>
+```
+
+![](images/numberbox-basic.png)
+
+### Add placeholder text
+
+XAML
+```XAML
+<NumberBox AutomationProperties.Name="simple NumberBox" Header="Enter equation:" PlaceholderText="A + B * C" />
+```
+
+![](images/numberbox-placeholder-text.png)
+
+### Enable calculation support
+
+XAML
+```XAML
+<NumberBox AutomationProperties.Name="NumberBox for simple calculations" 
+    Header="Enter equation:" 
+    PlaceholderText="A + B * C" 
+    AcceptsCalculation="True" />
+```
+
+![](images/numberbox-acceptscalculation.png)
+
+### Add increment and decrement stepping
+
+XAML
+```XAML
+<NumberBox AutomationProperties.Name="NumberBox for items in order" 
+    StepSize="0.1"
+    UpDownButtonsEnabled="True"
+    UpDownDragEnabled="True" />
+```
+
+![](images/numberbox-updownbuttons.png)
+
+### Add mininum and maximum values
+
+XAML
+```XAML
+<NumberBox AutomationProperties.Name="NumberBox for items in order" 
+    PlaceholderText="1" 
+    MinValue="1"
+    MaxValue="99" />
+```
+
+## Remarks
+<!-- Explanation and guidance that doesn't fit into the Examples section. -->
+
+<!-- APIs should only throw exceptions in exceptional conditions; basically,
+only when there's a bug in the caller, such as argument exception.  But if for some
+reason it's necessary for a caller to catch an exception from an API, call that
+out with an explanation either here or in the Examples -->
+
+## API Notes
+<!-- Option 1: Give a one or two line description of each API (type
+and member), or at least the ones that aren't obvious
+from their name.  These descriptions are what show up
+in IntelliSense. For properties, specify the default value of the property if it
+isn't the type's default (for example an int-typed property that doesn't default to zero.) -->
+
+<!-- Option 2: Put these descriptions in the below API Details section,
+with a "///" comment above the member or type. -->
+
+## API Details
+<!-- The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) -->
+
+```c++ 
+unsealed runtimeclass NumberBox : Windows.UI.Xaml.Controls.TextBox
+{
+    NumberBox();
+    
+    Double Value;
+    
+    Boolean AcceptsCalculation;
+    Boolean UpDownButtonsEnabled;
+    Boolean HyperDragEnabled;
+    Boolean HyperScrollEnabled;
+    
+    Double StepSize;
```

Comment:
Note that this doc must include rules to verify the StepSize relative to MaxValue and MinValue and what to do when excpetional values are provided.

Reply:
Agreed. I will leave this comment unresolved as a forum to discuss preference.

Reply:
@mrlacey, I have added this to the Behavioral Notes section with this commit: https://github.com/microsoft/microsoft-ui-xaml-specs/commit/164f9c67bcf6711606df6bc453c7da34bc16c116